pages/linux/whatis.md

```diff
@@ -10,6 +10,10 @@
 
 `whatis --long {{command}}`
 
+- Display descriptions for all commands matching a glob:
+
+`whatis --wildcard {{command_glob}}`
```

Comment:
Can we give concrete examples for both glob and regular_expression ? I think it will be easier for users to see something actual.